src/test/java/de/medizininformatikinitiative/torch/service/CrtdlProcessingServiceIT.java

```diff
@@ -165,15 +156,18 @@ void empty() throws JsonProcessingException {
     @Test
     void processReferences() {
         String jobId = "processwithrefs";
-        jobDir = resultFileManager.initJobDir(jobId).block();
+        Path jobDir = resultFileManager.initJobDir(jobId).block();
 
         Mono<Void> result = service.process(CRTDL_OBSERVATION_LINKED, jobId, List.of());
 
 
         Assertions.assertDoesNotThrow(() -> result.block());
-        try (DirectoryStream<Path> stream = Files.newDirectoryStream(jobDir)) {
-            boolean filesExist = stream.iterator().hasNext();
-            assertTrue(filesExist, "Job directory should contain files.");
+        try {
+            Assertions.assertNotNull(jobDir);
+            try (DirectoryStream<Path> stream = Files.newDirectoryStream(jobDir)) {
+                boolean filesExist = stream.iterator().hasNext();
+                assertTrue(filesExist, "Job directory should contain files.");
+            }
         } catch (IOException e) {
```

Comment:
please declare exceptions right on the test method. there is no need for custom logging here